src/replicache.test.ts

```diff
@@ -104,7 +97,7 @@ teardown(async () => {
   }
 
   for (const name of dbsToDrop) {
-    await clear(name);
+    indexedDB.deleteDatabase(name);
```

Comment:
This relies a bit on internal knowledge of how replicache treats `name`, but I guess it's ok. Was the prior method really slow or something?

Reply:
If we just delete all the keys there is a history left behind which impacts the hashes.